extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1422,48 +1417,20 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   startMeasuringAllVideoPerformance_() {
-    if (!this.mediaPerformanceMetricsService_.isPerformanceTrackingOn()) {
-      return;
-    }
-
     const videoEls = /** @type {!Array<!HTMLMediaElement>} */ (
       this.getAllVideos_()
     );
     for (let i = 0; i < videoEls.length; i++) {
-      this.startMeasuringVideoPerformance_(videoEls[i]);
-    }
-  }
-
-  /**
-   * @param {!HTMLMediaElement} videoEl
-   * @private
-   */
-  startMeasuringVideoPerformance_(videoEl) {
-    if (!this.mediaPerformanceMetricsService_.isPerformanceTrackingOn()) {
-      return;
+      this.mediaPerformanceTracker_.track(videoEls[i]);
     }
-
-    this.performanceTrackedVideos_.push(videoEl);
-    this.mediaPerformanceMetricsService_.startMeasuring(videoEl);
   }
 
   /**
-   * Stops measuring video performance metrics, if performance tracking is on.
-   * Computes and sends the metrics.
-   * @param {boolean=} sendMetrics
+   * @param {boolean=} sendMetrics true by default
    * @private
    */
-  stopMeasuringAllVideoPerformance_(sendMetrics = true) {
-    if (!this.mediaPerformanceMetricsService_.isPerformanceTrackingOn()) {
-      return;
-    }
-
-    for (let i = 0; i < this.performanceTrackedVideos_.length; i++) {
-      this.mediaPerformanceMetricsService_.stopMeasuring(
-        this.performanceTrackedVideos_[i],
-        sendMetrics
-      );
-    }
+  stopMeasuringAllVideoPerformance_(sendMetrics) {
+    this.mediaPerformanceTracker_.stop(sendMetrics !== false);
```

Comment:
This function seems to be useless when you can call directly `this.mediaPerformanceTracker_.stop`, so even more removal of code